opentelemetry-sdk/src/opentelemetry/sdk/_metrics/metric_exporter.py

```diff
@@ -0,0 +1,17 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Nit: Rename file to `_exporter.py`?

Reply:
Maybe, but the classes in this module would still be named `XMetricExporter`, so it is not very consistent :shrug:

Reply:
We should probably be consistent with logs and traces and have a `export` folder then?

Reply:
All right, added the `export` folder :v: